src/Components/Server/src/DependencyInjection/ServerSizeBlazorBuilderExtensions.cs

```diff
@@ -12,6 +12,29 @@ namespace Microsoft.Extensions.DependencyInjection
     /// </summary>
     public static class ServerSizeBlazorBuilderExtensions
     {
+        /// <summary>
+        /// Adds options to configure circuits.
+        /// </summary>
+        /// <param name="builder">The <see cref="IServerSideBlazorBuilder"/>.</param>
+        /// <param name="configure">A callback to configure <see cref="CircuitOptions"/>.</param>
+        /// <returns>The <see cref="IServerSideBlazorBuilder"/>.</returns>
+        public static IServerSideBlazorBuilder AddCircuitOptions(this IServerSideBlazorBuilder builder, Action<CircuitOptions> configure)
```

Comment:
Makes it a bit more discoverable. This is fairly undiscoverable now. How do we feel about `AddServerSideBlazor` optionally accepting a callback to configure `CircuitOptions`?

Reply:
I think that would be good. Its a bit jarring to see AddServer**Size**Blazor and the options type name to be different, but I can live with that.